zeppelin-web/pom.xml

```diff
@@ -120,34 +152,77 @@
           </execution>
 
           <execution>
-            <id>yarn install</id>
-            <goals>
-              <goal>yarn</goal>
-            </goals>
-            <configuration>
-              <arguments>install --no-lockfile</arguments>
-            </configuration>
+              <id>yarn add bower</id>
+              <goals>
+                  <goal>yarn</goal>
+              </goals>
+              <configuration>
+                <!--ignore-scripts-->
+                  <arguments>add bower --no-lockfile --no-bin-links</arguments>
```

Comment:
May be you know better... My main goal is to make developers free from local env for start development and build. Also i wanted give easier ability newbies try zeppelin. I don't care about how to UI developers will run these scripts locally. 
They can run this as they used to. I have just changed "postinstall" trigger in package.json because it interfered with "yarn install" from frontend-maven-plugin execution (yarn install). Do you want remove all changes in *.MD or remove all changes in pom and package.json?